src/CommitmentsV2/SFA.DAS.CommitmentsV2.Api/Startup.cs

```diff
@@ -63,6 +68,7 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerF
 
             app.UseHttpsRedirection()
                 .UseApiGlobalExceptionHandler(loggerFactory.CreateLogger("Startup"))
+                .UseUnauthorizedAccessExceptionHandler()
```

Comment:
I think this may be breaking one of the V2 integration tests.

Reply:
Hmm they run fine for me, are you using the latest `das-employer-config`? The schema has been updated to support the feature toggling in this PR.